src/components/Buttons/DiscordButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from "react"
+import { Button, ButtonProps } from "@chakra-ui/react"
+import { FaTwitter, IoLogoDiscord, IoLogoGithub } from "react-icons/all"
+import { ExternalLinkHref } from "../Navbar/types"
+
+const DiscordButton: FC<ButtonProps> = ({ ...props }) => {
+  return (
+    <Button
```

Comment:
Looks like we can use `ExternalButtonLink` component. Also, we repeat almost the same for each social media site. Maybe we can create something like:
```
const SocialMediaButton = ( { socialMediType: DISCORD|GITHUB|TELEGRAM } ) => {
 return <render the correct button based on the `socialMediType`>
}
```

and then in other component:
```
<SocialButton socialMediType={DISCORD} />
```

Just a thought but it won't probably work well with the cms until we add support for `mdx` files.